project/plugins.sbt

```diff
@@ -1,15 +1,9 @@
 resolvers += Resolver.url("fix-sbt-plugin-releases", url("http://dl.bintray.com/sbt/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "0.8.5")
-
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.0")
-
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.1")
-
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.0.1")
-
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
-
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
-
-addSbtPlugin("com.eed3si9n" % "sbt-unidoc" % "0.3.3")
+addSbtPlugin("com.typesafe.sbt"  % "sbt-git"       % "0.8.5")
+addSbtPlugin("com.github.gseitz" % "sbt-release"   % "1.0.3")
```

Comment:
My spider senses are saying that there was some issue with selecting the specific versions for two or more of these dependencies.  I suppose we wont find out until the next release, but it's just a head's up.  I thought I would have added a comment somewhere here, but I didn't, so who's to say.